packages/binding-http/src/http-server.ts

```diff
@@ -606,14 +646,6 @@ export default class HttpServer implements ProtocolServer {
             );
         });
 
-        // Set CORS headers
-        if (this.httpSecurityScheme !== "NoSec" && req.headers.origin) {
-            res.setHeader("Access-Control-Allow-Origin", req.headers.origin);
-            res.setHeader("Access-Control-Allow-Credentials", "true");
-        } else {
-            res.setHeader("Access-Control-Allow-Origin", "*");
-        }
-
```

Comment:
Note: I was surprised first "why" we no longer need CORS but the code has moved to "common"

Reply:
Yeah, it is a tricky change, but in a nutshell: CORS management depends on the security schema used on that resource. Since now we don't have one single security strategy for the whole server, we need to handle CORS on each affordance and understand which security mechanisms is used for that specific thing (in the future for that specific form -> URL path).

Reply:
Do you have any ideas to improve future devs about this choice? 🤔

Reply:
I just wanted to highlight it for other reviewers. I am fine with your decision 👍